metadata-ingestion/src/datahub/ingestion/source/powerbi/powerbi.py

```diff
@@ -419,11 +440,51 @@ def chart_custom_properties(dashboard: PowerBiAPI.Dashboard) -> dict:
             dashboard_key_mcp,
         ]
 
+        if self.__config.extract_workspaces_to_containers:
+            container_mcp = self.add_urn_to_container(
+                self.gen_workspace_key(dashboard.workspace_name),
+                dashboard_urn,
+                Constant.DASHBOARD,
+            )
+            list_of_mcps.append(container_mcp)
+
         if owner_mcp is not None:
             list_of_mcps.append(owner_mcp)
 
         return list_of_mcps
 
+    def gen_workspace_key(self, workspace_name: str) -> PlatformKey:
+        return WorkspaceKey(
+            workspace=workspace_name,
+            platform=self.__config.platform_name,
+        )
+
+    @staticmethod
+    def add_urn_to_container(
+            container_key: KeyType, entity_urn: str, entity_type: str
+    ) -> MetadataChangeProposalWrapper:
+        container_urn = builder.make_container_urn(
+            guid=container_key.guid(),
```

Comment:
Could you please add `get_urn_part` method in Workspace data class. It will make code consistent with other data-classes like User , Page, Table ...